view/src/worker.rs

```diff
@@ -178,8 +177,7 @@ impl Worker {
             if inbound_transaction_ids.contains(&tx_id)
                 || transaction
                     .spent_nullifiers()
-                    .iter()
-                    .any(|nf| spent_nullifiers.contains(nf))
+                    .any(|nf| spent_nullifiers.contains(&nf))
```

Comment:
This is a fix because the nullifiers now are a copying iterator rather than a method returning a `Vec` of references.